docs/testing/clear-cache.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Clear the Office cache
+description: Learn how to clear the Office cache on your computer.
+ms.date: 12/31/2019
+localization_priority: Priority
+---
+
+# Clear the Office cache
+
+You can remove an add-in that you've previously sideloaded on Windows, Mac, or iOS by clearing the Office cache on your computer. 
+
+Additionally, if you make changes to your add-in's manifest (for example, update file names of icons or text of add-in commands), you should clear the Office cache and then re-sideload the add-in using updated manifest. Doing so will allow Office to render the add-in as it's described by the updated manifest.
+
+## Clear the Office cache on Windows
+
+To clear the Office cache on Windows, delete the contents of the folder `%LOCALAPPDATA%\Microsoft\Office\16.0\Wef\`.
+
+## Clear the Office cache on Mac
+
+[!include[additional cache folders on Mac](../includes/mac-cache-folders.md)]
```

Comment:
There's an intro sentence to this Mac include file that's repetitive in this context. However, it's needed for the other locations at which it's included. Not sure what's the best approach, but I wanted to at least point out this issue.

Reply:
Good point; I'll note this as something we can potentially address in the future.